static/src/js/components/OrderStatus/OrderStatusItem.js

```diff
@@ -71,7 +71,8 @@ export class OrderStatusItem extends PureComponent {
     const { type: accessMethodType } = accessMethod
 
     // TODO: Add a second value and refresh at different intervals for the different types of orders
-    const { orderStatusRefreshTime } = getApplicationConfig()
+    const { orderStatusRefreshTime, env } = getApplicationConfig()
```

Comment:
So in `static.config.json` we have `clientId` which is an object in the `application` JSON. I am not sure if this ticket is supposed to pass that along? We should make sure.

Reply:
Those don't seem to reflect the running environment, but the ticket asked for `edsc-sit, edsc-uat, edsc-prod`
```json
    "clientId": {
      "background": "eed-PORTAL-ENV-serverless-background",
      "client": "eed-PORTAL-ENV-serverless-client",
      "lambda": "eed-PORTAL-ENV-serverless-lambda"
    }
```

Reply:
We have logic that replaces `PORTAL` and `ENV` with the appropriate values. That `PORTAL` gets replaced with the deployments defaultPortal, in our case that will be `edsc`, but if someone else deploys EDSC the idea is that they would setup their own default portal which would separate their requests from ours.

`sharedUtils/getClientId.js` is where the replacements happens.

To be consistent with how those clientIds are used over the rest of the application we should use `getClientId().client` since this would be the React client-side application sending the request. `background` is used in background lambdas and `lambda` is used for the rest of our application's lambdas

Reply:
Implemented with latest commit